packages/turf-boolean-touches/types.ts

```diff
@@ -1,9 +0,0 @@
-import * as helpers from "@turf/helpers";
-import { booleanTouches } from "./index.js";
-
-const pt = helpers.point([0, 0]);
-const line = helpers.lineString([
-  [0, 0],
-  [10, 10],
-]);
-booleanTouches(pt, line);
```

Comment:
I think the point of these was to help guard against API breaks in the types? Kind of like what DefinitelyTyped does. I'm not sure that that's a super compelling argument here with a relatively basic amount of syntax expressed in this file. The `last-checks` step is grabbing the code in the examples and checking that the example compiles. Thoughts? (I do kind of want each package to test its own example, documentation generation, etc).

Reply:
Do you mean like a form of API locking between major versions? I thought so too at first. However, I could easily narrow the signature of booleanTouches to Feature<Point>, Feature<LineString> and this test would still pass. As is, I don't think they really protect us much.

> I do kind of want each package to test its own example, documentation generation, etc).

Definitely. That is still the case. The `tsc --noEmit` in each package is checking to see if the module compiles. We don't need to actually call our method to have tsc examine it though. Just importing seems to be enough.

We can add a bunch of example invocations if we want. All we're doing though is proving we know how to call our own library. It's not testing the type safety of the implementation any better.